cmake/Platform/Common/PackagingCodeSign.cmake

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+message(STATUS "Executing package signing...")
+
+function(ly_sign_binaries in_path sign_type)
+    file(REAL_PATH "${CPACK_SOURCE_DIR}/.." _root_path)
+    unset(_signing_command)
+
+    if(WIN32 OR CYGWIN)        
```

Comment:
a cleaner way to do this, is just have two implementations of this functions, one for windows and one for linux in Platform/<platform>/ and make the scripts include one or the other
Can be converted later

Reply:
Sure, shouldn't be an issue to break it up